src/portalbackend/PortalBackend.DBAccess/Models/ConnectorData.cs

```diff
@@ -36,7 +36,9 @@ public record ConnectorData(
     Guid? HostId,
     string? HostCompanyName,
     Guid? SelfDescriptionDocumentId,
-    string? DocumentName);
+    string? DocumentName,
+    TechnicalUserData? TechnicalUsers
```

Comment:
please change to technicalUser

```suggestion
    TechnicalUserData? TechnicalUser
```